tests/testutils/telemetry/pdata_logs.go

```diff
@@ -0,0 +1,139 @@
+// Copyright Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package telemetry
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+)
+
+// PDataToResourceLogs returns a ResourceLogs item generated from plog.Logs content.
+// At this time histograms and summaries aren't supported.
```

Comment:
```suggestion
```

Reply:
sry, there is a lot of c/p in these. when traces or more matching rule capabilities are added moving to a common loader will be helpful.